frontend/public/components/utils/simple-tab-nav.tsx

```diff
@@ -17,7 +17,7 @@ class SimpleTab extends React.PureComponent<SimpleTabProps> {
     const {active, title} = this.props;
     const className = classNames('co-m-horizontal-nav__menu-item', {'co-m-horizontal-nav-item--active': active});
     return <li className={className}>
-      <a href="#" onClick={this.onClick}>{title}</a>
+      <button onClick={this.onClick}>{title}</button>
```

Comment:
type="button"

If we even use these tabs in a form, we don't want it to submit the form.